### ISSUE TITLE
Orca: the optimizer_creator and loss_func shouldn't be required in PytorchPysparkEstimator.predict() 

### ISSUE THREAD

**Issue by zpeng1898** [intel-analytics/BigDL#7161]
When predicting xshards using `PytorchPysparkEstimator`, an error occurs showing that the `optimizer_creator` and `loss` parameters are required. But in fact, they shouldn't be required in predict() after loading the pretrained model.

Code:
![image](https://user-images.githubusercontent.com/42887453/210506654-fe409c15-2a65-440b-ab19-38b67c85c4e4.png)

Problem:
![image](https://user-images.githubusercontent.com/42887453/210506909-9c4afa37-8a32-4bda-a434-9de0a4b32d52.png)

The error disappears after adding the optimizer_creator and loss to the estimator:
![image](https://user-images.githubusercontent.com/42887453/210506866-d16290e6-cb51-47c8-99f7-8587a2b84022.png)

**Comment by hkvision** [comment]
@sgwhat Please take a look.

**Comment by zpeng1898** [comment]
![image](https://user-images.githubusercontent.com/42887453/210511292-609d1a53-fa51-4d94-9915-6e862df60407.png)
![image](https://user-images.githubusercontent.com/42887453/210511352-62d156bd-0d65-4262-bc8a-4ae5cfb788e1.png)

**Comment by hkvision** [comment]
https://github.com/intel-analytics/BigDL/pull/6988/files#diff-9ecfa460f1e73f063bab468a4717117b6d390ba93ce1e5e7c13e49b94533d05fL141
@leonardozcm Please take a look.

**Comment by leonardozcm** [comment]
https://github.com/intel-analytics/BigDL/pull/7168 track here

**Comment by hkvision** [comment]
This should have been fixed. Closing it.
